packages/components/src/slot-fill/index.native.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { omit } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import BaseSlot from './slot';
+import Fill from './fill';
+import Provider from './provider';
+
+export { Fill, Provider };
+
+export function Slot( props ) {
+	return <BaseSlot { ...omit( props, 'bubblesVirtually' ) } />;
```

Comment:
This needs more thinking. The first step could be to simply throw an error when the version that bubbles virtually is used. There are some differences between those two APIs that need to be bridged first.

Reply:
I don't think we should throw an error personally as this will allow us to have the same consumer code for mobile and web.